### ISSUE TITLE
Frozen Backbone Implementation

### ISSUE THREAD

**Issue by HaFred** [alexrich021/3dvnet#11]
Hi Alex @alexrich021 , thank you for sharing this great work. I have a question about the frozen implementation during training, which is described in Sec. 4.1 in the article. All I can found about the "frozen implementation on MnasNet backbone" is as below.
https://github.com/alexrich021/3dvnet/blob/8ea11dca65964d952986fe33ef7b922f0aa0c300/mv3d/lightningmodel.py#L272

However, according to the [PyTorch documentation](https://pytorch.org/docs/stable/notes/autograd.html#locally-disable-grad-doc), `.eval()` only fixes the running statistics in batchnorm and dropout, but does not disable the gradient computations for MnasNet backbone during BackProp.

I also run the training script to see whether MnasNet backbone weights are really frozen or not. It turns out that with `.eval()`, the backbone weights are still updating for each time ` training_step()` is called. Could you kindly elaborate on the frozen backbone here? Does it really mean to fix the weights in the backbone during training? Thank you very much.

**Comment by alexrich021** [comment]
Hi Fred, thanks for the interest! Great catch, the proper method is to change `3dvnet/mv3d/lightningmodel.py` line 286 to
```python
{'params': self.mvsnet.feat_shrinker.parameters()},
```
during training to freeze the weights of the MnasNet backbone and to use
```python
{'params': self.mvsnet.parameters()},
```
during finetuning. I'll fix that this weekend. During training of models in the paper, the MnasNet backbone was definitely frozen for initial training and only unfrozen during finetuning. Thanks for finding this and I hope this helps.
